src/style.css

```
@@ -1 +1,7 @@
-/* styles go here */
+.input {
+  margin-bottom: 10px;
+}
```

```suggestion
.input:not(:last-child) {
  margin-bottom: 10px;
}
```
It may fix your tests

---

Ok. Tests are passed. Thanks.